### ISSUE TITLE
Support alt text for author photos

### ISSUE THREAD

**Issue by matthiasott** [matthiasott/webmention#9]
The plugin should pull the alt text for author photos.

The mf2 parser already returns an `alt` attribute:
```
{
  "items": [{
    "type": ["h-card"],
    "properties": {
      "name": ["Mr. Example"],
      "photo": [{
        "value": "http://example.org/photo.png",
        "alt": ""
      }]
    }
  }],
  "rels": {},
  "rel-urls": {}
}
```

This alt text should be added when the asset is being saved.
